frontend/src/common/components/Header/Header.tsx

```diff
@@ -12,6 +12,11 @@ const StyledContainer = styled.header`
   position: fixed;
   width: 48rem;
   height: 5.7rem;
+  border-bottom: 1px solid ${({ theme }) => theme.LINE.REGULAR};
+  background: ${({ theme }) => theme.BG.LIGHT + 'CC'};
```

Comment:
[질문] theme.BG.LIGHT + 'CC' 에서 'CC'가 뭐야?? 왜 더하고 있어?

---

찾아보니까 투명도를 결정하던데 혹시 그럼 rgb로 통일하는 건 어떨까? 🤔

```suggestion
  background: rgb(${({ theme }) => theme.BG.LIGHT} r g b / 80%);
```

[참고자료](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/rgb#syntax)

Reply:
피그마 디자인 색상과 theme으로 지정해둔 색이 달랐는데, 색상에서 투명도만 변경하면 될 것 같아서 ai 에게 투명도 추천을 받았는데 꼼꼼하게 확인 못하고 넣었네 😢

섬세한 리뷰 좋습니다~

Reply:
새롭게 알게 되는 게 있어서 코드 리뷰 재밌고 좋은데요~~~ 😊